go/master/client.go

```diff
@@ -104,25 +103,44 @@ func NewClient(opts ...func(*Client) error) (*Client, error) {
 		if err != nil {
 			return nil, err
 		}
-
 	}
-
+	c.passID = 0
+	c.ch = make(chan record, c.bufSize)
+	// FIXME: connection is created asyncrosly in monitorMaster go routine,
+	//        ensure the connection is ready for use before calling c.addClient.
+	time.Sleep(time.Second)
```

Comment:
Yep. Connection is created in a go-routine `monitorMaster`, immediate use of the `conn` may due to error.